openapi/openapi.yaml

```diff
@@ -1080,6 +1080,35 @@ paths:
               schema:
                 $ref: "#/components/schemas/PythonPackageMetadataResponseError"
 
+  /python/get-imported-packages:
```

Comment:
the endpoint actually does the opposite, `/get-package-from-imports` cc @fridex

Reply:
fixed in 0ada6fd042629810b21dcec94b4be45bd7b93e6b